Makefile

```diff
@@ -113,23 +113,33 @@ submodules:
 	@git submodule sync
 	@git submodule update --init --recursive
 
-.PHONY: manifests cobertura reviewable submodules fallthrough test-integration
+# This is for running out-of-cluster locally, and is for convenience. Running
+# this make target will print out the command which was used. For more control,
+# try running the binary directly with different arguments.
+run: go.build
+	@$(INFO) Running Crossplane locally out-of-cluster . . .
+	@# To see other arguments that can be provided, run the command with --help instead
+	$(GO_OUT_DIR)/$(PROJECT_NAME) --debug
+
+.PHONY: manifests cobertura reviewable submodules fallthrough test-integration run
 
 # ====================================================================================
 # Special Targets
 
-define CROSSPLANE_HELP
+define CROSSPLANE_MAKE_HELP
 Crossplane Targets:
     manifests          Generate manifests e.g. CRD, RBAC etc.
     cobertura          Generate a coverage report for cobertura applying exclusions on generated files.
     reviewable         Ensure a PR is ready for review.
     submodules         Update the submodules, such as the common build scripts.
 	
 endef
-export CROSSPLANE_HELP
+# The reason CROSSPLANE_MAKE_HELP is used instead of CROSSPLANE_HELP is because the crossplane
+# binary will try to use CROSSPLANE_HELP if it is set, and this is for something different.
```

Comment:
Good catch here! Comment was very useful for my understanding for this PR, do you think we should leave it in when merging?

Reply:
I typically do, to help future maintainers (including myself) understand the reason the code is the way that it is. My usual goal for code is for it to be self-documenting, which I think of as:

* The code is simple enough that reading the code explains **what** the code is doing (with comments for any necessary exceptions)
* Comments or docs exist that explain **why** the code is this way; I think of this as being able to get into the mindset of the author, so that future maintainers are able to extend and work with the code in the way that the original author intended (when they want to)

In this case, since there are no tests per se, a comment like this could also help to avoid potential regressions in the future.

Reply:
I can get on board with that! Appreciate your attention to detail and developer experience :)